README.md

```diff
@@ -12,6 +12,20 @@ Requisites
 
 - [NodeJS](https://nodejs.org) (at least v0.10)
 - [Git](https://git-scm.com)
+- The following executables should be correctly installed in your shell environment:
+  - `ssh`: Any recent version of the OpenSSH ssh client (required by `resin sync` and `resin ssh`)
+  - `rsync`: >= 2.6.9 (required by `resin sync`)
+
+##### Windows Support
+
+`resin sync` and `resin ssh` have not been thoroughly tested on the standard Windows cmd.exe shell. We recommend using bash (or a similar) shell, like Bash for Windows 10 or Git Bash.
```

Comment:
@lekkas Should it link to Git for Windows actually? I'm guessing that's what "Git Bash" really is, from your comment on the issue, right? https://git-for-windows.github.io/